commands/make/prldfile.ts

```diff
@@ -8,7 +8,10 @@
  */
 
 import BaseCommand from './_base.js'
-import { args } from '../../modules/ace/main.js'
+import { args, flags } from '../../modules/ace/main.js'
+import { AppEnvironments } from '@adonisjs/application/types'
```

Comment:
```suggestion
import type { AppEnvironments } from '@adonisjs/application/types'
```

Reply:
Definitely need that eslint rule ahah. Thanks dude